src/Sulu/Bundle/ContentBundle/Resources/public/js/components/smart-content/main.js

```diff
@@ -129,7 +128,6 @@ define([], function() {
             hideCategories: false,
             hideTags: false,
             hideSortBy: false,
-            hidePresentAs: false,
```

Comment:
I wouldn't remove this line, since it might be possible that we want to make this configureable.